lib/adapters/linter-push-v2-adapter.ts

```diff
@@ -113,13 +113,13 @@ export default class LinterPushV2Adapter {
     if (path != null) {
       const diagnosticCodes = this._diagnosticCodes.get(path);
       if (diagnosticCodes != null) {
-        return diagnosticCodes.get(getCodeKey(range, text));
+        return diagnosticCodes.get(getCodeKey(range, text)) || null;
```

Comment:
Maps return `undefined` when a key has no value, so I'm coercing it to `null` to conform to the existing signature